Makefile

```diff
@@ -1,3 +1,9 @@
+# generate version number from the latest git tag; if the topmost commit is
+# tagged (at release time), just use that; if there are commits after that,
+# append a ".x" suffix to indicate that's a development snapshot
+# if there is no tag (yet), use 0 as version
+RELEASEVER=$(shell (git describe --exclude '*jenkins*' || echo 0) | sed 's/-[0-9]\+-g.*/.x/')
```

Comment:
This still needs to be fixed to conform to semver, if there are new commits it really should bump the patch number and add -rc so that it evaluates after the previous version.

Reply:
These tarballs are not ever supposed to be public, so how precisely to mark them as "not for public use" is fairly theoretical - However, as you insist: Just incrementing the last digit would be too presumptuous IMHO. After an actual 3.0 release someone might want to release a 3.0.1, so an intermediate version needs to be smaller than that, and not be called 3.1-rc (which would be greater than 3.0.1). Also, these are certainly not release candidates.

So how about appending "-alpha.0" instead? That should be compatible with semver, never sort after the next official version number, and does not make some of the strong semver assumptions, i. e. that should be compatible with rpm version comparison, too. (I hope - e. g. dpkg disallows dashes in version numbers, need to try with rpm).

Reply:
Yep, same with RPM:
```
rpmbuild -bs \
  --define "_sourcedir `pwd`" \
  --define "_srcrpmdir `pwd`" \
  --define "releasever 0.1-alpha.0" \
  welder-web.spec
error: line 6: Illegal char '-' (0x2d) in: Version: 0.1-alpha.0
```
As this is mostly just an test-internal vehicle, do we really need to bother?

Reply:
As long as there's no chance of releasing these I guess it is ok.